### ISSUE TITLE
misp-modules keeps restarting

### ISSUE THREAD

**Issue by ghost** [DCSO/MISP-dockerized#80]
I have just done a fresh install of MISP-dockerized 1.1.1, on a Digital Ocean Ubuntu 18.04 Docker droplet, and `misp-modules` restarts over and over again with the following log.
```
No module named 'requests'
2019-08-01 15:52:30,989 - misp-modules - INFO - Launch MISP modules server from current directory.
2019-08-01 15:52:30,991 - misp-modules - INFO - Helpers failed cache.py due to Redis not running or not installed. Helper will be disabled.
2019-08-01 15:52:30,993 - misp-modules - INFO - MISP modules osqueryexport imported
2019-08-01 15:52:30,993 - misp-modules - INFO - MISP modules testexport imported
2019-08-01 15:52:30,994 - misp-modules - INFO - MISP modules threat_connect_export imported
2019-08-01 15:52:30,995 - misp-modules - WARNING - MISP modules pdfexport failed due to No module named 'pymisp'
2019-08-01 15:52:30,996 - misp-modules - INFO - MISP modules threatStream_misp_export imported
2019-08-01 15:52:30,997 - misp-modules - INFO - MISP modules nexthinkexport imported
2019-08-01 15:52:30,997 - misp-modules - INFO - MISP modules cef_export imported
2019-08-01 15:52:30,998 - misp-modules - INFO - MISP modules cisco_firesight_manager_ACL_rule_export imported
2019-08-01 15:52:30,999 - misp-modules - INFO - MISP modules liteexport imported
2019-08-01 15:52:31,000 - misp-modules - WARNING - MISP modules goamlexport failed due to No module named 'pymisp'
2019-08-01 15:52:31,002 - misp-modules - INFO - MISP modules mispjson imported
2019-08-01 15:52:31,011 - misp-modules - INFO - MISP modules email_import imported
2019-08-01 15:52:31,012 - misp-modules - WARNING - MISP modules csvimport failed due to No module named 'pymisp'
2019-08-01 15:52:31,013 - misp-modules - INFO - MISP modules ocr imported
2019-08-01 15:52:31,017 - misp-modules - WARNING - MISP modules goamlimport failed due to No module named 'pymisp'
2019-08-01 15:52:31,018 - misp-modules - INFO - MISP modules threatanalyzer_import imported
2019-08-01 15:52:31,019 - misp-modules - INFO - MISP modules testimport imported
2019-08-01 15:52:31,021 - misp-modules - WARNING - MISP modules joe_import failed due to No module named 'pymisp'
2019-08-01 15:52:31,023 - misp-modules - WARNING - MISP modules vmray_import failed due to No module named 'requests'
2019-08-01 15:52:31,023 - misp-modules - INFO - MISP modules cuckooimport imported
2019-08-01 15:52:31,024 - misp-modules - WARNING - MISP modules openiocimport failed due to No module named 'pymisp'
2019-08-01 15:52:31,025 - misp-modules - WARNING - MISP modules stiximport failed due to No module named 'pymisp'
yara is missing, use 'pip3 install -I -r REQUIREMENTS' from the root of this repository to install it.
2019-08-01 15:52:31,028 - misp-modules - INFO - MISP modules yara_query imported
2019-08-01 15:52:31,029 - misp-modules - WARNING - MISP modules joesandbox_query failed due to No module named 'jbxapi'
2019-08-01 15:52:31,030 - misp-modules - WARNING - MISP modules securitytrails failed due to No module named 'dnstrails'
dnspython3 is missing, use 'pip install dnspython3' to install it.
```
I tried to build an image from `dcso/misp-dockerized-misp-modules:2`, running `pip3 install -I -r REQUIREMENTS` and `pip3 install dnspython3`, but the `docker-build` fails with:

```
Collecting opencv-python==4.1.0.25 (from -r REQUIREMENTS (line 41))
  Could not find a version that satisfies the requirement opencv-python==4.1.0.25 (from -r REQUIREMENTS (line 41)) (from versions: )
No matching distribution found for opencv-python==4.1.0.25 (from -r REQUIREMENTS (line 41))
You are using pip version 18.1, however version 19.2.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
The command '/bin/sh -c cd /usr/local/src/misp-modules && pip3 install -I -r REQUIREMENTS' returned a non-zero code: 1
```
Is there anything that I can do about that?

Thanks.

**Comment by 8ear** [comment]
Hi @carey-at-xq,
Sorry for my late answer.
You have two options at the current time. use our dev image: `dcso/misp-dockerized-misp-modules:3-dev` or wait until I released #70 . (I hope in this week if all is working).
The best is go to your config/config.env file and add `MISP_MODULES_CONTAINER_TAG=3-dev`

**Comment by ghost** [comment]
Thanks, Max. Will do.
